src/module-config/module-config.ts

```diff
@@ -22,27 +22,52 @@ export function getConfig(moduleName) {
   }
   const schema = schemas[moduleName];
 
-  // Create a config object composed of all the defaults
-  const defaultConfig = R.map(R.prop("default"), schema);
-
   // Merge all of the configs provided for moduleName
+  const mergeDeepAll = R.reduce(R.mergeDeepRight, {});
```

Comment:
For tree shaking to work, you should not do `import * as R from 'ramda'`, but instead import only the methods you're using. The reason is that tree shaking is about removing code that is not imported into the bundle, but importing * means you need all of it. The way you can test if tree shaking is just by running `npm run build` and checking bundle size in the dist directory. You can also use https://github.com/webpack-contrib/webpack-bundle-analyzer to verify exactly which parts of the ramda library are being included.

Reply:
I tried this out locally and it looks like I was wrong. My understanding of tree shaking is correct, but I was unaware of what babel-plugin-ramda was doing. From [their readme](https://github.com/megawac/babel-plugin-ramda):

> This plugin is a transform to remove unused ramda dependencies, without forcing the user to cherry pick methods manually.